inc/managers/class-form-manager.php

```diff
@@ -116,7 +116,7 @@ public function display_form(): void {
 			'
 		<div v-cloak data-wu-app="%s" data-state="%s">
 			<ul class="wu-p-4 wu-bg-red-200 wu-m-0 wu-list-none" v-if="errors.length">
-				<li class="wu-m-0 wu-p-0" v-for="error in errors">{{ error.message }}</li>
+				<li class="wu-m-0 wu-p-0" v-for="error in errors" v-html="error.message"></li>
```

Comment:
What is the purpose of this change?

Reply:
<img width="1070" height="1355" alt="image" src="https://github.com/user-attachments/assets/1fcd4f9f-e303-433d-b1f5-e51342a36459" />

Reply:
Beautiful! :+1: